src/d3a/models/strategy/load_hours.py

```diff
@@ -275,6 +287,9 @@ def _one_sided_market_event_tick(self, market, offer=None):
             self.log.exception("An Error occurred while buying an offer")
 
     def _double_sided_market_event_tick(self, market):
+        """
+        Define the behavior of the device on TICK events in double-sided markets (post bids).
+        """
```

Comment:
This is a perfect example for a redundant docstring IMO. Leave it please, but really, the function name is already telling what is done.
Also, the `(post bids)` addition is not really true, right? It updates the bids. Right?

Reply:
I think you're right, also from the moment that the function is private so it doesn't need to necessarily have a docstring.